src/libraries/System.Diagnostics.DiagnosticSource/tests/ActivityTests.cs

```diff
@@ -1428,6 +1428,86 @@ public void TestIsAllDataRequested()
             Assert.Equal(1, a1.Tags.Count());
         }
 
+        [Fact]
+        public void TestTagObjects()
+        {
+            Activity activity = new Activity("TagObjects");
+            Assert.Equal(0, activity.Tags.Count());
+            Assert.Equal(0, activity.TagObjects.Count());
+
+            activity.AddTag("s1", "s1").AddTag("s2", "s2").AddTag("s3", null);
+            Assert.Equal(3, activity.Tags.Count());
```

Comment:
is null value also treated as string, and is returned along with `Tags`?

Reply:
Yes, in old AddTag(string, string) allowed null.